### PR TITLE
V0.7 merge back to master

### PR DESCRIPTION
**Pull request by chicm-ms**

### CODE REVIEW

File: deployment/deployment-pipelines.yml [hunk 1]
```diff
@@ -182,35 +182,78 @@ jobs:
...
+        python -m twine upload -u $env:TESTPYPI_USER -p $env:TESTPYPI_PWD --repository-url https://test.pypi.org/legacy/ dist/*
+      }
+      else{
+        Write-Host 'uploading release package to pypi...'
+        python -m twine upload -u $env:PYPI_USER -p $env:PYPI_PWD dist/*
+      }
+    condition: eq( variables['upload_package'], 'true')
+    displayName: 'upload nni package to pypi/testpypi'
+
+- job: 'Build_upload_nni_win_amd64'
```
Comment (squirrelsc) [inline deployment/deployment-pipelines.yml]: Can this part share more code with above x86?

File: deployment/pypi/README.md [hunk 2]
```diff
@@ -47,15 +47,14 @@ This is the PyPI build and upload tool for NNI project.
...
-  tar
   ```
 
 * __How to build__
-
+ 
+  parameter `version_os` is used to build for Windows 64-bit or 32-bit. 
   ```bash
-  powershell ./install.ps1
+  powershell ./install.ps1 -version_os [64/32]
```
Comment (squirrelsc) [inline deployment/pypi/README.md]: The windows version can be detected automatically, it's better to handle the logic ourselves, instead of asking users.

Comment (chicm-ms) [reply deployment/pypi/README.md]: Hi Chi, this is a merge back PR, could you  please open an issue, so we can review and fix it later?   Normally we do not change things in merge back PR.

**Comment by QuanluZhang** [comment]
please pass integration test

File: deployment/deployment-pipelines.yml (hunk 1 above)
Comment (chicm-ms) [reply deployment/deployment-pipelines.yml]: Please open an issue for this.  It will be reviewed and fixed later.

**Comment by chicm-ms** [comment]
> please pass integration test

The failed integration tests are not caused by this PR.

**Pull request merged**